packages/react-composites/src/composites/CallComposite/styles/CallPage.styles.ts

```diff
@@ -2,39 +2,20 @@
 // Licensed under the MIT license.
 
 import { IStackItemStyles, IStackStyles, IStyle, mergeStyles } from '@fluentui/react';
+import { controlBarContainerStyles } from './CallControls.styles';
 
-/**
- * @private
- */
-export const callControlsStyles: IStackItemStyles = {
-  root: {
-    width: '100%'
-  }
-};
+const VIDEO_GALLERY_Z_INDEX = 1;
+const CONTROL_BAR_Z_INDEX = VIDEO_GALLERY_Z_INDEX + 1;
+const NOTIFICATION_CONTAINER_Z_INDEX = Math.max(CONTROL_BAR_Z_INDEX, VIDEO_GALLERY_Z_INDEX) + 1;
```

Comment:
@prprabhu-ms I think because these are sibling components we don't need to have the z-index higher than the media gallery sub component, however what errors/ufd things should I test here to make sure nothing is broken?

Reply:
You can test the recording/transcription permission banner, and/or the "you're muted" notification.

They're all in the same container.

Reply:
Confirmed these:
![image](https://user-images.githubusercontent.com/2684369/141521070-fcb2cda9-905f-40e3-b3d3-1739d37eddde.png)
![image](https://user-images.githubusercontent.com/2684369/141521082-185245bb-a5af-43e5-afdb-538d1676d372.png)